### ISSUE TITLE
FR: Google Play Store inclusion

### ISSUE THREAD

**Issue by svenstaro** [readest/readest#1001]
**Does your feature request involve difficulty for you to complete a task? Please describe.**
I think that for better accessibility, this app should be included in the Google Play Store. It's already offered in the Apple App Store and it's hopefully going to be [available in F-Droid](https://github.com/readest/readest/issues/627) soonish.

**Describe the solution you'd like**
Add it to the Google Play Store.
